docs/fleet/provision.md

```diff
@@ -135,6 +135,16 @@ The Viam Agent is installed as a `systemd` service named `viam-agent`.
 
   For more information, see [Viam Agent management](https://github.com/viamrobotics/agent#management).
 
+### View Viam Agent logs
+
+The Viam Agent writes log messages to the [Viam app](https://app.viam.com/).
+These messages can be found on the [**LOGS** tab](/cloud/machines/#logs) of your machine's page.
+
+Log messages are only sent when your machine is online and connected to the internet.
+If your machine is offline, log messages are queued, and are sent to the Viam app once your machine reconnects to the internet.
+
+These log messages include when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warning encountered during operation.
```

Comment:
```suggestion
These log messages include logs of when `viam-server` is stopped and started, the status of agent subsystems, and any errors or warning encountered during operation.
```
[opt] I don't love my suggestion, but existing wording is grammatically iffy/awk

Reply:
This one is too many `logs`